list/list.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>list</title>
+    <link href="list.css" rel="stylesheet" type="text/css" />
+    <link href="https://fonts.googleapis.com/css2?family=Fruktur&family=Inter:wght@200;400;600;700&family=Mulish&family=Nunito+Sans:wght@300&family=Ubuntu:wght@300&family=Zen+Antique+Soft&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="heading">
+        <div class="component-title">List</div>
+        <div class="component-description">HTML Lists are used to specify lists of information. All lists may contain one or more list elements. </div>
+
+    </header>
+    <main class="container">
+        <!-- spaced list -->
+        <div class="side-heading">
+            Spaced List/Normal List
+        </div>
+        <div class="side-heading-description">Spaced list or normal list are used in work places to create a space between list items</div>
+        <div class="lists">
+            <div class="spaced-list">
+                <ul>
+                    <li class="spaced-list-items">Price </li>
+                    <li class="spaced-list-items">Category</li>
+                    <li class="spaced-list-items">Ratings</li>
+                    <li class="spaced-list-items">Sort products</li>
+                </ul>
+
+            </div>
+        </div>
+        <div class="code-snippet">
+            <embed src="https://carbon.now.sh/embed?bg=rgba%28189%2C16%2C224%2C1%29&t=vscode&wt=none&l=htmlmixed&width=608.25&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=15.5px&lh=177%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cul%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced-list-items%2522%253EPrice%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced-list-items%2522%253ECategory%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced-list-items%2522%253ERatings%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced-list-items%2522%253ESort%2520products%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Ful%253E"
+                style="width: 608px; height: 222px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin" type="">
```

Comment:
Try keeping the styling in a separate file. Here, you can try and move the iFrame styling to one of the common stylesheets.